Assets/Scripts/SceneSwitcher.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SceneSwitcher : MonoBehaviour
+{
+    public void switchScene()
+    {
+        SceneManager.LoadScene("Menu");
+    }
+}
```

Comment:
I think we can remove this script as I said in comment above.

Reply:
Done